src/modules/31-variables/components/CreateEditVariable/__tests__/CreateEditVariable.test.tsx

```diff
@@ -10,13 +10,16 @@ import { noop } from 'lodash-es'
 import { act } from 'react-dom/test-utils'
 import { render, waitFor } from '@testing-library/react'
 import { TestWrapper } from '@common/utils/testUtils'
-
 import { clickSubmit, fillAtForm, InputTypes } from '@common/utils/JestFormHelper'
-import { useCreateVariable } from 'services/cd-ng'
+import { VariableSuccessResponseWithData } from '@variables/pages/variables/__tests__/mock/variableResponse'
+import { useCreateVariable, useUpdateVariable } from 'services/cd-ng'
 import CreateEditVariable from '../CreateEditVariable'
 
 jest.mock('services/cd-ng')
 const useCreateVariableMock = useCreateVariable as jest.MockedFunction<any>
+const useUpdateVariableMock = useUpdateVariable as jest.MockedFunction<any>
+const updateVariable = jest.fn().mockImplementationOnce(() => ({ status: 'SUCCESS' }))
+useUpdateVariableMock.mockImplementation(() => ({ mutate: updateVariable }))
```

Comment:
You can simplify this a wee bit using `mockReturnValueOnce` and `mockReturnValue` respectively.
```suggestion
const updateVariable = jest.fn().mockReturnValueOnce({ status: 'SUCCESS' })
useUpdateVariableMock.mockReturnValue({ mutate: updateVariable })
```

Fine as-is though 👍🏻